common/_overrides.scss

```diff
@@ -7,20 +7,44 @@
     background-color: $hover-bg !important;
     color: $hover-color !important;
     border-color: $hover-border-color !important;
+    & span {
+      color: $hover-color !important;
+    }
+  }
+
+  & span {
+    color: $color !important;
   }
 }
 
 // Set the default color of links
-a:not(.btn),
+a:not(.btn, .help-link),
 a:not(.btn):hover,
 a:not(.btn):focus,
 a:visited:not(.btn):hover,
-a:visited:not(.btn):focus,
-a:not(.help-link) {
+a:visited:not(.btn):focus {
   color: $link-color;
 }
 
-.btn-primary {
+// Set color for course nav bar in Learning MFE
+.course-tabs-navigation a.nav-link{
```

Comment:
Stylelint: missing space.
Please use [stylelint](https://stylelint.io/) on this file.